tests/parametric/test_process_discovery.py

```diff
@@ -63,7 +65,9 @@ def test_metadata_content(self, test_library, library_env):
             # assert tracer_metadata["hostname"]
             # TODO(@dmehala): how to get the version?
             # assert tracer_metadata["tracer_version"] ==
```

Comment:
Hi @cbeauchesne , do you know how could I retrieve the version of the tracer here?

Reply:
`context.library.version`. Though, it may contains some difference, versions of dd-trace repo are very heterogeneous,  I had to do some trick to support all of them, meaning that the value you'll see in `context.library.version`  can differ from the one reported.

Reply:
But at least, checking major/minor/patch should be good enough. Hacks are mostly on pre-release / built parts